cmd/pipeline/deploy.go

```diff
@@ -126,10 +126,56 @@ func (pc *Command) ExecuteDeployPipeline(ctx context.Context, opts *DeployOption
 			return fmt.Errorf("failed to load okteto context '%s': %v", okteto.Context().Name, err)
 		}
 
-		_, err = configmaps.Get(ctx, pipeline.TranslatePipelineName(opts.Name), opts.Namespace, c)
+		cfgName := pipeline.TranslatePipelineName(opts.Name)
+
+		cfg, err := configmaps.Get(ctx, cfgName, opts.Namespace, c)
 		if err == nil {
-			oktetoLog.Success("Skipping repository '%s' because it's already deployed", opts.Name)
-			return nil
+			if cfg != nil && cfg.Data != nil {
```

Comment:
I've haven't added tests of the function `ExecuteDeployPipeline` because it does have  dependencies which make it hard to add them, and it'd require a refactor.

I suggest having a follow-up issue for this PR to schedule for next sprint for example, to refactor this function and make it easier to test

Reply:
Maybe we can encapsulate this new code into a function. Probably that makes this new code testable without having to test `ExecuteDeployPipeline` function

Reply:
ok I've tried the idea of extracting the new code into a function, however it became very complicated too, especially testing all the stuff that is a dependency like `configmaps.WaitForStatus` and `pc.waitUntilRunning` without having to refactor a huge amout.

It would probably be quicker and safer to takle this as an ad-hoc refactor, rewriting some code from scratch..